Taskfile.yml

```diff
@@ -45,7 +45,6 @@ tasks:
       - gofumpt -w .
       - go mod tidy
       - golangci-lint run --timeout 5m --fix
-      - nilaway -fix -test=false -include-pkgs='github.com/opslevel/kubectl-opslevel' ./...
```

Comment:
Removing because `nilaway -fix` isn't functional yet, and even if it was we would want to make developers handle `nil` situations carefully and manually.